subset/security/password/test_password

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash -e
```

Comment:
How is this different than the subset/pentests/test_brute code?  I'm just trying to make sure we're not ending up with duplicate stuff -- should test_brute be removed?

Reply:
Yes, test_brute should be removed as it is not doing anything useful. We kept it so far only for "educational" reasons.